Formula/qtkeychain.rb

```diff
@@ -0,0 +1,26 @@
+class Qtkeychain < Formula
+  desc "Qt API for storing passwords securely in the keychain"
+  homepage "https://github.com/frankosterfeld/qtkeychain"
+  url "https://github.com/frankosterfeld/qtkeychain/archive/v0.8.0.tar.gz"
+  sha256 "b492f603197538bc04b2714105b1ab2b327a9a98d400d53d9a7cb70edd2db12f"
+
+  depends_on 'cmake' => :build
+  depends_on "qt"
+
+  def install
+    # We need to set QT_TRANSLATIONS_DIR, else it tries to install translations
+    # into Qt's directory which fails
+    system "cmake", ".", "-DQT_TRANSLATIONS_DIR=#{prefix}/translations", *std_cmake_args
+    system "make", "install"
+    bin.install "testclient"
```

Comment:
i guess it's only for development. i only install it to be able to run the brew test. suggestions?